modules/compatcheck/dagger.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "compatcheck",
+  "sdk": "go",
+  "dependencies": [
+    {
+      "name": "dagger-dev",
+      "source": "github.com/dagger/dagger@386e4d5597cd41598292ef6b463b7d762866679a"
```

Comment:
I *think* we can do this with `../../.dagger`.

Note: yeah it's a bit weird that this would be in `modules` but not be called by `.dagger`. That's annoying, but I think we have a similar issue where we might want to be able to do this for the sdk `runtime` modules. I think it's fine for now to have this in `modules/` for now :tada: We can always move it later.